server/txSender/dataFormatter.go

```diff
@@ -68,8 +69,19 @@ func (df *dataFormatter) createRegisterBridgeOperationsData(bridgeData *sovereig
 	registerBridgeOpData := []byte(registerBridgeOpsPrefix +
 		"@" + hex.EncodeToString(bridgeData.AggregatedSignature) +
 		"@" + hex.EncodeToString(bridgeData.Hash))
+	registerBridgeOpData = append(registerBridgeOpData, hashesHexEncodedArgs...)
 
-	return append(registerBridgeOpData, hashesHexEncodedArgs...)
+	bridgeDataArgs := []byte(
+		"@" + hex.EncodeToString(bridgeData.PubKeysBitmap) +
+			"@" + hex.EncodeToString(uint32ToBytes(bridgeData.Epoch)))
```

Comment:
Do we really need the `Epoch`?
Do we really need a to keep all the maps <epoch, validators> in the contract storage?

Reply:
If we send the new validators set at epoch change, we can just update the same storage, we don't need the epoch

Reply:
I see why you need epoch, maybe you have to send unconfirmed operations in the next epoch, which were signed by the validators from previous epoch

Reply:
I see now why we need to keep the history. Maybe there are unconfirmed operations sent in the epoch 1, but were signed by the validators in epoch 0

Reply:
it is better to keep for at least 2-3 epochs.